Dockerfile

```diff
@@ -30,7 +30,8 @@ RUN \
     syslog-ng-stomp \
     syslog-ng-tags-parser \
     syslog-ng-xml \
-    py3-syslog-ng && \
+    py3-syslog-ng \
+    paho-mqtt-c && \
```

Comment:
Please sort these alphabetically

Reply:
fixed

Reply:
P is before S

Reply:
Fixed, but just to let you know that py3-syslog-ng was already at the end and not on top. means someone (tm) missed sorting before and nobody took care...